modules/nf-core/dada2/meta.yml

```diff
@@ -0,0 +1,55 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "dada2"
+description: write your description here
+keywords:
+  - microbial taxonomy
+  - amplicon sequence variant (asv)
+  - amplicon sequencing
+  - error correction
+  - denoising
+
+tools:
+  - "dada2":
+      description: "DADA2: High-resolution sample inference from Illumina amplicon data"
+      homepage: "https://benjjneb.github.io/dada2/"
+      documentation: "https://www.bioconductor.org/packages/3.6/bioc/manuals/dada2/man/dada2.pdf"
+      tool_dev_url: "https://github.com/benjjneb/dada2"
+      doi: "https://doi.org/10.1038/nmeth.3869"
+      licence: LGPL version 3
```

Comment:
I think this is the correct way to write it:
licence: ["LGPL >=3"]

Reply:
Ok, I'll trust you on this one